ansible/roles/content-snapshot-indexer/tasks/main.yml

```diff
@@ -0,0 +1,129 @@
+---
+- name: Create content-snapshot directory
+  file:
+    path: '{{ content_snapshot_path }}'
+    state: directory
+    mode: 0777
+  become: yes
+  become_user: "{{ analytics_user }}"
```

Comment:
This code is redundant. We have already used become module in playbook level. So this is not required and it will fail since analytics user does not have sudo access.